model-optimizer/mo/utils/cli_parser.py

```diff
@@ -845,6 +845,23 @@ def get_freeze_placeholder_values(argv_input: str, argv_freeze_placeholder_with_
     return placeholder_values, input_node_names
 
 
+def parse_dimension(dim: str):
+    if '..' in dim:
+        numbers_reg = r'[0-9]+'
```

Comment:
numbers_reg = r'^[0-9]+$'.
please add a test case to handle incorrect case: [6754fg..23ed]

Reply:
Done.